assoc-aggregate/assoc-aggregate.wdl

```diff
@@ -517,90 +516,71 @@ task sbg_prepare_segments_1 {
 			return chrom_num
 
 		def pair_chromosome_gds(file_array):
+			# Corresponds with pair_chromosome_gds() in CWL
```

Comment:
Since the CWL is a moving target, it might be a good idea to reference which commit in CWL you are writing the WDL to emulate.